make-file-list.sh

```diff
@@ -140,6 +140,16 @@ mingw-w64-$PACMAN_ARCH-zstd"
 	return $res
 }
 
+has_pacman_package () {
```

Comment:
There seem to be no callers for this.

Reply:
Gaah, you're right, I forgot to adjust the code below that function to add two intended callers! [Fixed](https://github.com/git-for-windows/build-extra/compare/701c6302d172010b25e414bcb49f12e90bc275a7..6769038c55aed5f067fec4309f96cc1ee0f7bbba)